src/Features/Core/Diagnostics/AnalyzerHelper.cs

```diff
@@ -69,8 +79,18 @@ public static Action<Diagnostic> GetAddExceptionDiagnosticDelegate(DiagnosticAna
             Func<Exception, DiagnosticAnalyzer, bool> continueOnAnalyzerException,
             CancellationToken cancellationToken)
         {
-            var addExceptionDiagnostic = GetAddExceptionDiagnosticDelegate(analyzer, hostDiagnosticUpdateSource, project);
+            var addExceptionDiagnostic = analyzer.GetAddExceptionDiagnosticDelegate(hostDiagnosticUpdateSource, project);
             return AnalyzerExecutor.Create(compilation, analyzerOptions, addDiagnostic, addExceptionDiagnostic, continueOnAnalyzerException, cancellationToken);
         }
+
+        private static VersionStamp GetProviderVersion(string path)
```

Comment:
Minor: You mean GetAnalyzerVersion? Some day we will get rid of term "provider" from our IDE diagnostic service :)